.travis.yml

```diff
@@ -3,7 +3,4 @@ script:
   - curl https://raw.githubusercontent.com/shapesecurity/CLA/master/cla-travis.sh | bash
   - npm run build
 node_js:
-  - "0.12"
-  - "0.11"
-  - "0.10"
-  - iojs
+  - "5.3"
```

Comment:
At a guess, the issue is with npm2 vs npm3. I haven't had time to investigate properly yet.